CHANGELOG.md

```diff
@@ -0,0 +1,61 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+This project adheres to [Semantic Versioning](http://semver.org/).
+
+## [Unreleased][unreleased]
+### Added
+- Support for Basic Auth
+  - ENV variables `COURIER_AUTH_USERNAME` and `COURIER_AUTH_PASSWORD` OR set with params
+- Token Auth using ENV variables
+  - `COURIER_AUTH_TOKEN`
+- Base URL parameter (with default of api.courier.com)
+  - ENV variable `COURIER_BASE_URL`
+- Support for Lists API by @jrweingart
+  - `POST /send/list` with `client.lists.send` method
+  - `GET /profiles/{recipient_id}/lists` with `client.lists.find_by_recipient_id` method
+  - `GET /lists` with `client.lists.list` method
+  - `GET /lists/{list_id}` with `client.lists.get` method
+  - `PUT /lists/{list_id}` with `client.lists.put` method
+  - `DELETE /lists/{list_id}` with `client.lists.delete` method
+  - `PUT /lists/{list_id}/restore` with `client.lists.restore` method
+  - `GET /lists/{list_id}/subscriptions` with `client.lists.get_subscriptions` method
+  - `PUT /lists/{list_id}/subscriptions` with `client.lists.put_subscriptions` method
+  - `PUT /lists/{list_id}/subscriptions/{recipient_id}` with `client.lists.subscribe` method
+  - `DELETE /lists/{list_id}/subscriptions/{recipient_id}` with `client.lists.unsubscribe` method
+
+- Support for Profiles API by @jrweingart
+  - `GET /profiles/{recipient_id}` with `client.profiles.get` method
+  - `GET /profiles/{recipient_id}/lists` with `client.profiles.get_subscriptions` method
```

Comment:
This route is covered twice. Once in Profiles and another in List. Did we mean to do that?